cmd/commands/commands_test.go

```diff
@@ -191,3 +190,139 @@ func TestReplaceCustomData(t *testing.T) {
 		})
 	}
 }
+
+// TestReplaceAndAppendScid tests whether chan_id is replaced with scid and
+// scid_str in the JSON console output.
+func TestReplaceAndAppendScid(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name     string
+		data     string
+		expected string
+	}{
+		{
+			name:     "no replacement necessary",
+			data:     "foo",
+			expected: "foo",
+		},
+		{
+			name: "valid json with replacement",
+			data: "{\"foo\":\"bar\",\"chan_id\":\"" +
```

Comment:
Nit: I think when creating json strings it is always good to use ` instead of escaping all those special characters. For future changes.